src/ranges.js

```diff
@@ -318,11 +318,35 @@ const getSegmentBufferedPercent = function(startOfSegment,
   return percent;
 };
 
+/**
+ * Gets a human readable string for a TimeRange
+ *
+ * @param {TimeRange} range
+ * @returns {String} a human readable string
+ */
+const printableRange = (range) => {
+  let str = '';
+
+  if (!range || !range.length) {
+    return str;
+  }
+
+  for (let i = 0; i < range.length; i++) {
+    if (i > 0) {
+      str += ', ';
```

Comment:
You can avoid this if you make `str` and array pushing each start+end and return `str.join(',')`. (It's also better for performance, not that these ranges will ever be large enough for that to matter)